server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -108,11 +108,11 @@
   /**
    * Options for the garbage collector.
    */
-  static class GCOpts extends ServerOpts {
-    @Parameter(names = {"-v", "--verbose"},
-        description = "extra information will get printed to stdout also")
-    boolean verbose = false;
-    @Parameter(names = {"-s", "--safemode"}, description = "safe mode will not delete files")
+  static class GCOpts extends ConfigOpts {
+    @Parameter(names = {"-s", "--safemode"}, validateWith = DeprecatedOption.class,
```

Comment:
Will this cause this to show up in any automated help documentation? It might be easier to leave this in the ServerOpts file for now, with your new warning.

Reply:
Yes it is printed with a warning saying that it is deprecated:
 ```
@Parameter(names = {"-s", "--safemode"}, validateWith = DeprecatedOption.class,
        description = "This option is DEPRECATED and replaced with property gc.safemode=true. "
            + "Safe mode will list files but not delete them")
```
This was  never in the ServerOpts - only specific to GC.